.github/pull_request_template.md

```diff
@@ -0,0 +1,37 @@
+
+<!--- Provide a general summary of your changes in the Title above -->
+
+#### Description
+<!--- Describe your changes in detail -->
+
+#### Related Issue
+<!--- This project only accepts pull requests related to open issues -->
+<!--- If suggesting a new feature or change, please discuss it in an issue first -->
+<!--- If fixing a bug, there should be an issue describing it with steps to reproduce -->
+<!--- Please link to the issue here: -->
+
+#### Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+
+#### How Has This Been Tested?
+<!--- Please describe in detail how you tested your changes. -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+#### Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
```

Comment:
I had considered something similar and then realized people *usually* think their bug is severe because it impacts them. I think as project moderators we should decide the severity of issues based on the impact to our users (%users affected, %algos affected, ect)